lib/ansible/modules/files/blockinfile.py

```diff
@@ -104,28 +102,32 @@
     description:
       - 'This flag indicates that filesystem links, if they exist, should be followed.'
     version_added: "2.1"
+notes:
+  - This module supports check mode.
+  - When using 'with_*' loops be aware that if you do not set a unique mark the block will be overwritten on each iteration.
```

Comment:
Yes, it was actually moved to the same location as the other modules (below the options).
At first I added the section, and then a test failed because `notes:` was twice in the dictionary :-(